### ISSUE TITLE
Issues with Tooltip docs

### ISSUE THREAD

**Issue by glen-84** [zurb/foundation-sites#11054]
From the example provided:

```html
<span
    data-tooltip
    aria-haspopup="true"
    class="has-tip"
    data-disable-hover="false"
    tabindex="1"
    title="Fancy word for a beetle.">
```

**aria-haspopup="true"**

According to [this doc](https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup), this attribute is not applicable to tooltips:

> A tooltip is not considered to be a popup in this context.

**class="has-tip"**

This class is added automatically, so it seems unnecessary unless you want to ensure that the CSS is applied even if the JS fails to execute (but then the tooltip, at least the Foundation one, would not show anyway).

**data-disable-hover="false"**

This is the default value, so it's unnecessary to include it.

**tabindex="1"**

It would be good to document this. I don't really understand its purpose.

**Comment by ncoden** [comment]
Hi @glen-84. Thanks for the issue. I opened 2 PRs: #11056 and #11057.

**Comment by glen-84** [comment]
Does anyone know why the `tabindex` is added?

**Comment by glen-84** [comment]
I know – I'm asking specifically about what it does in this case. 🙂

**Comment by DanielRuf** [comment]
It makes it accessible for screenreaders and people with assistive solutions.

**Comment by glen-84** [comment]
I've found the answer. Without the `tabindex`, the element is not "focusable". With the attribute, the element can receive focus, and the tooltip is displayed.

**Comment by DanielRuf** [comment]
Right, that's what I've meant with "assistive solutions" like the OSD (onscreen keyboard) and other things like keyboard users (specific person groups).

**Comment by ncoden** [comment]
I guess this can be closed.
